developer/src/kmc-convert/src/convert-utils.ts

```diff
@@ -0,0 +1,83 @@
+
+/*
+ * Keyman is 2025 copyright (C) SIL International. MIT License.
+ *
+ * Created by S. Schmitt on 2025-08-24
+ *
+ * functions for kmc-convert utils
+ *
+ */
+
+/**
+ * @brief  function to convert a numeric character reference or a unicode value to a unicode character e.g. &#x63 -> c;  U+1F60E -> 😎
+ * @param  inputString the value that will converted
+ * @return a unicode character like 'c', 'ሴ', '😎' or undefined if inputString is not recognized
+ */
+export function convertToUnicodeCharacter(inputString: string): string {
```

Comment:
I am uncomfortable with this function. We shouldn't be mixing inputs like `U+1234` with plain characters and XML named entities. What are its use cases?